### DISCUSSION TITLE
How to always show signature help when typing?

### DISCUSSION THREAD

**Discussion by idelice** [folke/noice.nvim#621] [General]
Imagine you have a method 'Foo()'.

Foo takes some params.

Now, when you start typing for the first time it shows the signature help, and then vanishes for ever.

How do I keep it on as long as my cursor is inside the params/method parantheses?

**Comment by xdanep** [comment]
Did you solve this? I have the same question. Untill now i was using lsp_signature.nvim, but these two can't coexist.

**Reply by idelice** [reply]
See: https://github.com/folke/noice.nvim/discussions/621#discussioncomment-10854950

**Comment by radutomy** [comment]
I am also looking for the same thing

**Reply by idelice** [reply]
I never actually fixed it myself.

Somehow, it just started working, but if you want to manually enable signature help in insert mode, add this keymap:

```lua
vim.keymap.set("i", "<c-s>", "<cmd>lua vim.lsp.buf.signature_help()<CR>", { desc = "Signature help" })
```
